pages/job-category/index.js

```diff
@@ -0,0 +1,157 @@
+import Link from "next/link";
+import { styled } from "styled-components";
```

Comment:
See previous comment. 

```suggestion
import styled from "styled-components";
```